src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -110,6 +111,14 @@
 (def OIDMappings
   {schema/Str schema/Keyword})
 
+(def default-allow-subj-alt-names
+  false)
+
+(schema/defn ^:always-validate
+initialize-ca-config
+  [{:keys [allow-subject-alt-names] :or {allow-subject-alt-names default-allow-subj-alt-names} :as ca-data}]
```

Comment:
I think the `:keys` and `:or` here are redundant.

Reply:
Is this not saying to get the allow-subject-alt-names key, and if it doesnt exist, to get the default allow-subject-alt-names?

Reply:
Yeah, but isn't that basically what `(merge defaults ca-data)` does?